test/test_support.ts

```diff
@@ -7,7 +7,9 @@ import main = require('../lib/main');
 import path = require('path');
 import ts = require('typescript');
 
-export type StringMap = { [k: string]: string };
+export type StringMap = {
```

Comment:
let's do this as part of an other PR. I haven't touch this code (it only has been automatically reformated)